logstash-core-event/lib/logstash/event.rb

```diff
@@ -106,7 +106,7 @@ def clone
 
   public
   def to_s
-    self.sprintf("#{timestamp.to_iso8601} %{host} %{message}")
+    "#{timestamp.to_iso8601} #{self.sprintf("%{host} %{message}")}"
```

Comment:
We were not hit bad that before since we did not cache the template :(
The bad thing this is probably a logging statement somewhere.